### PR TITLE
Mentioning Selenium Manager in docs

### PR DESCRIPTION
**Pull request by diemol**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description
Selenium 4.6 will include beta 1 of Selenium Manager, which simplifies how user can
get started into Selenium. Therefore, we have modified the sections where instructions
to configure drivers are provided to mention that Selenium Manager will take care of 
setting the driver for Chrome, Edge, and Firefox.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/grid/getting_started.en.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ aliases: [
 1. Prerequisites
     * Java 11 or higher installed
     * Browser(s) installed
-    * Browser driver(s) [installed and on the `PATH`]({{< ref "../webdriver/getting_started/install_drivers.md#2-the-path-environment-variable" >}})
+    * Browser driver(s)
```
Comment (titusfortner) [inline website_and_docs/content/documentation/grid/getting_started.en.md:18]: These shouldn't be multiple bullets, and "getting started" should be minimal explanation. recommend:

```suggestion
    * Browser driver(s) [installed and on the `PATH`]({{< ref "../webdriver/getting_started/install_drivers.md#2-the-path-environment-variable" >}}) (if using Selenium 4.5 or lower)
```

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.en.md [hunk 2]
```diff
@@ -23,9 +23,24 @@ This section explains the basic requirements for getting started with the differ
...
+{{< badge-version version="4.6" >}}
+
+Selenium Manager helps you to get a working environment to run Selenium out of the box. Beta 1
+of Selenium Manager will configure the drivers for Chrome, Firefox, and Edge if they are not 
+found on the `PATH`. In addition, in .NET it is limited to Windows only. Future releases of
+Selenium Manager will support macOS and Linux for .NET, and eventually download browsers if
+necessary.
+
+Selenium Manager is shipped as part of Selenium 4.6, and it will be used when no driver is
+found for Chrome, Firefox, or Edge. No extra configuration is needed.
```
Comment (titusfortner) [inline website_and_docs/content/documentation/webdriver/getting_started/install_drivers.en.md]: this is kind of restating line 33.

**Review by titusfortner — COMMENTED** [review]
Looks good!

File: website_and_docs/content/documentation/grid/getting_started.en.md (hunk 1 above)
Comment (titusfortner) [reply website_and_docs/content/documentation/grid/getting_started.en.md:18]: (oh that showed up as an auto commit option; you'd have to also delete the next lines if you accept this.)

Comment (diemol) [reply website_and_docs/content/documentation/grid/getting_started.en.md:18]: I'd prefer to leave it more explicit in the docs.

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.en.md (hunk 2 above)
Comment (diemol) [reply website_and_docs/content/documentation/webdriver/getting_started/install_drivers.en.md]: Good catch, thank you.

**Pull request merged**